backend/tournesol/lib/public_dataset.py

```diff
@@ -26,7 +26,8 @@ def get_comparisons_data(poll_name: str) -> QuerySet:
             entity_2.uid AS uid_b,
             comparisoncriteriascore.criteria,
             comparisoncriteriascore.weight,
-            comparisoncriteriascore.score
+            comparisoncriteriascore.score,
+            EXTRACT('epoch' FROM DATE_TRUNC('week', datetime_add)) AS week_timestamp
```

Comment:
@lfaucon is it what you expect? for a video I uploaded on my local setup this week, it returns `1674432000.0` which stands for `Monday, January 23, 2023 12:00:00 AM`

Reply:
I think we can aim for a more human readable value.

As defined by [ISO 8601](https://fr.wikipedia.org/wiki/ISO_8601) we can represent the date by the string `yyyy-mm-dd`.

The following line should do the trick:

```sql
DATE(datetime_add) AS week

-- or
DATE(datetime_add) AS comparison_week
```

Reply:
> @lfaucon is it what you expect? for a video I uploaded on my local setup this week, it returns `1674432000.0` which stands for `Monday, January 23, 2023 12:00:00 AM`

Weird, right now is the week of Monday Jan 30, 2023 right?

Reply:
my bad, the comparisons were done last week and not this week so it makes sense

@GresilleSiffle don't we want to hide the day to preserve privacy? Let's say I add a comparison today (2023-02-01), we would expect `2023-01-30` in the export, right?

Reply:
> @GresilleSiffle don't we want to hide the day to preserve privacy? Let's say I add a comparison today (2023-02-01), we would expect `2023-01-30` in the export, right?

Correct

Reply:
Sorry I didn't express myself correctly =/

I thought that using `DATE` could help to convert a timestamp to a human readable date, and yes using `datetime_add` as argument is not correct as it will leak the real comparison date.